homeassistant/config_entries.py

```diff
@@ -1302,6 +1256,23 @@ async def async_init(
                 translation_domain=HOMEASSISTANT_DOMAIN,
             )
 
+        if context.get("source") == SOURCE_RECONFIGURE and any(
+            flow
+            for flow in self.hass.config_entries.flow.async_progress_by_handler(
+                handler,
+                match_context={"entry_id": context["entry_id"]},
+                include_uninitialized=True,
+            )
+            if flow["context"].get("source") == SOURCE_REAUTH
```

Comment:
Ok, so we now allow multiple parallel reconfigure flows. Do we allow that for option flows?

Reply:
I think so yes.
I couldn't find any check in `OptionsFlowManager` or elsewhere.